File: Webpack/src/js/Model/Destination.ts
/**
 *
 *
 * @export
 * @class Destination
 */
export class Destination {
    public name: string;
    public type: string;
    public routeIdx: number;
    public time: string;
    public rtTime: string;
    public date: string;
    public rtDate: string;
    public track: string;
    public rtTrack: string;

    constructor(name: string, type: string, time: string, date: string,
                routeIdx?: number, track?: string, rtTime?: string, rtDate?: string, rtTrack?: string) {
        this.name = name;
        this.type = type;
        this.time = time;
        this.date = date;

        this.routeIdx = routeIdx;
        this.track = track;
        this.rtTime = rtTime;
        this.rtDate = rtDate;
        this.rtTrack = rtTrack;
    }
}

File: nvrl8-ws/Model/destination.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace nvrl8_ws.Model
{
    public class Destination
    {
        public string Name{ get; set; }
        public string Type { get; set; }
        public int RouteIdx { get; set; }
        public string Time { get; set; }
        public string RtTime { get; set; }
        public string Date { get; set; }
        public string RtDate { get; set; }
        public string RtTrack { get; set; }
        public string Track { get; set; }

        public Destination(string name, string type, string time, string date,  string track, string rtTime = null, int routeIdx = 0, string rtDate = null, string rtTrack = null)
        {
            Name = name;
            Type = type;
            RouteIdx = routeIdx;
            Time = time;
            RtTime = rtTime;
            Date = date;
            RtDate = rtDate;
            RtTrack = rtTrack;
            Track = track;
        }

        public override string ToString()
        {
            return $"{nameof(Name)}: {Name}, {nameof(Type)}: {Type}, {nameof(RouteIdx)}: {RouteIdx}, {nameof(Time)}: {Time}, {nameof(RtTime)}: {RtTime}, {nameof(Date)}: {Date}, {nameof(RtDate)}: {RtDate}, {nameof(RtTrack)}: {RtTrack}, {nameof(Track)}: {Track}";
        }
    }
}
